6MI0800226/week1.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#pragma warning(disable: 4996)
+struct transition
+{};
+
+
+//zad 2
+struct pair
+{
+    int firstInt;
+    int secondInt;  
+};
+void inputIntsOfPair(pair& pair1)
+{
+    std::cout << "First integer: ";
```

Comment:
btw - it's great that you've decided to provide some meaningful prompts for the user :partying_face: